src/Microsoft.OData.Edm/Validation/EdmErrorCode.cs

```diff
@@ -1388,67 +1388,57 @@ public enum EdmErrorCode
         /// </summary>
         AnnotationApplyToNotAllowedAnnotatable = 400,
 
-        #region JSON
+        /// <summary>
+        /// Invalid $Key value.
+        /// </summary>
+        InvalidKeyValue,
 
         /// <summary>
-        /// Invalid JSON
+        /// Invalid mulitple entity container in one schema
         /// </summary>
-        JsonInvalid,
+        SchemaCannotHaveMoreThanOneEntityContainer,
 
         /// <summary>
-        /// The specified version number is not valid.
+        /// Invalid $Collection value.
         /// </summary>
-        JsonInvalidVersionNumber,
+        InvalidCollectionValue,
+
+        #region JSON
 
         /// <summary>
-        /// Invalid JSON value
+        /// Invalid JSON
         /// </summary>
-        JsonInvalidValue,
+        InvalidJson,
 
         /// <summary>
         /// Invalid JSON value kind
         /// </summary>
-        JsonUnknownElementValueKind,
+        UnknownJsonElementValueKind,

```

Comment:
>UnknownJsonElementValueKind [](start = 8, length = 27)

Can this just be "UnknownElementKind"?

Reply:
Changed to UnknowElementValueKind. That's related to ValueKind

---
In reply to: [445801156](https://github.com/OData/odata.net/pull/1829#discussion_r445801156) [](ancestors = 445801156)